bin/s2nd.c

```diff
@@ -42,9 +43,15 @@
 #include "common.h"
 
 #include "utils/s2n_safety.h"
+#include "tls/s2n_connection.h"
```

Comment:
Why do you need this? We should try to stick to public APIs + "soon to be" public APIs.